poi-ooxml/src/test/java/org/apache/poi/openxml4j/opc/compliance/TestOPCComplianceCoreProperties.java

```diff
@@ -295,4 +305,50 @@ void testNoCoreProperties_saveInPlace() throws Exception {
         }
         assertTrue(tmp.delete());
     }
+
+    /**
+     * Test M4.2 rule not enforced when appropriate OPCComplianceFlag is set.
+     */
+    @Test
+    void testDisableEnforceCompatibilityMarkup() throws IOException {
+        testComplianceFlagOverridePreventsException(
+                "DoNotUseCompatibilityMarkupFAIL.docx",
```

Comment:
these files - should they be in the PR or are they already in the repo? It doesn't look right that they have no relative path info.

Reply:
if these are new files that need to be added to the repo (test-data dir and it has subdirs based on file type), it will probably break the poi-integration tests which run automatic tests on all files in test-data dir and invalid files will need to be explicitly skipped

Reply:
These files already exist and are already being used in the neighboring tests. The existing tests in this file check that we _do_ enforce compliance. My additional tests confirm that when a specific compliance setting (or settings) are turned off, the file then parses without error.

I'm using the same `OpenXML4JTestDataSamples.openSampleStream` method the other tests use, which I believe handles the actual prefixing, etc.

I ran all of the tests with gradle test, and everything passed on my machine including all the `poi-integration` tests, so I think we're okay? Happy to adjust anything if I missed something.